expat/configure.ac

```diff
@@ -307,6 +307,12 @@ AS_IF([test "x${DOCBOOK_TO_MAN}" != x -a "x$with_docbook" != xno],
 
 AM_CONDITIONAL(WITH_DOCBOOK, [test "x${DOCBOOK_TO_MAN}" != x])
 
+dnl updating _EXPAT_TARGET variable to effect the package name in expat.pc file (issue #361)
+AC_SUBST(_EXPAT_TARGET, ["$PACKAGE_NAME"])
+AS_CASE(["${host_os}"],
+  [win32],[AC_SUBST(_EXPAT_TARGET, ["libexpat"])])
+AM_COND_IF([UNICODE], [AC_SUBST(_EXPAT_TARGET, ["expatw"])])
```

Comment:
I was working on the very same thing — just pushed as #363.
I'm happy to go with commits by you rather than me but we should go with constant `expat` and no `expatw` here because distros patch expat.pc in case of Autotools and need for expatw using `unsigned char`. So this can and should be a lot simpler. Autotools is on the way out of here so we just need to keep things alive. The next release may even be with CMake alone, if #358 goes far enough by then.

Reply:
Updated the change by removing expatw constant.